front/lib/api/assistant/actions/websearch.ts

```diff
@@ -0,0 +1,49 @@
+import type { AgentActionSpecification, Result } from "@dust-tt/types";
+import { Ok } from "@dust-tt/types";
+import type { WebsearchConfigurationType } from "@dust-tt/types/dist/front/assistant/actions/websearch";
+
+import type { Authenticator } from "@app/lib/auth";
+
+function websearchActionSpecification(arg0: {
```

Comment:
This needs to follow the pattern introduced in https://github.com/dust-tt/dust/pull/5215 (cc @lasryaric)

Reply:
There is a card in go/todo that I had created specifically to switch to the new pattern. Would it be fine to you if I did the refactor for the new pattern right after in a following PR? (I think i'll be faster and get it right better this way) :pray:

Reply:
I'd rather push to have it know given that it's less code and in the follow up PRs we can focus on the core code rather than some decoration 🙃.

Reply:
I meant I would do it right after, immediately following :) So there would be no follow up PRs that would rely on the legacy pattern
Would that be fine?

Reply:
ok, ok, i'm doing it :upside_down_face: